apis/v1alpha1/tcproute_types.go

```diff
@@ -51,42 +51,14 @@ type TCPRouteRule struct {
 	// if **any** one of the matches is satisfied.
 	//
 	// +optional
-	Matches []TCPRouteMatch `json:"matches"`
+	// +kubebuilder:validation:MaxItems=8
```

Comment:
// +kubebuilder:validation:MinItems=0

Reply:
Do we need to specify minItems=0 if that's already the default?